app/cafe.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
```

Comment:
According to the checklist, it's recommended to use parentheses when importing multiple classes/modules. Here, you're using parentheses, but you're also using a backslash to break the lines. You can remove the backslashes and just rely on the parentheses to keep the code cleaner and more Pythonic. Here's how you can do it: 
python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```